apps/src/types/redux.ts

```diff
@@ -58,3 +59,7 @@ export interface RootState {
   teacherRubric: TeacherRubricState;
   teacherSections: TeacherSectionState;
 }
+
+interface ManageStudentsState {
+  studentData: object[];
```

Comment:
Could this be more specific than just `object`?

Reply:
it definitely could. This is many handled in a reducer that's a `.js` file.  I figured I would let the future dev who migrates it to TypeScript to properly flesh it out. I will add some more fields though.